packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx

```diff
@@ -206,6 +206,15 @@ export class ViewerTransformWidget {
   @Prop()
   public rotationHandleScalar = 1;
 
+  /**
+   * Specifies the delay, in milliseconds, to update the transform after interactions with the widget.
+   *
+   * This delay is used to group events happening in quick succession and results in smoother
+   * widget movement.
+   */
+  @Prop()
+  public interactionDebounce = 75;
```

Comment:
Nit: I think it'd be more accurate to call this an `interactionThrottle`, since a debounce implies that we'll only perform the transform after 75ms of inactivity, where as this functionality prevents more than one transform per 75ms

Reply:
Good point - updated!